components/Stats.js

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<ul className='font-lexend text-5xl decoration-black font-semibold absolute left-6 bottom-11 leading-15 flex items-center text-center space-x-32'>
```

Comment:
Stay away from the large margins `absolute left-6 bottom-11 leading-15` and `space-x-32`, they often times cause your code to break because they rely on certain properties of the screen that we cannot guarantee each time